File: Data Structures & Algorithms/length-of-last-word/submission-2.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        count = 0
        i = len(s)-1
        while(s[i] == ' ' and i >= 0):
            i = i-1
        
        j = i

        while(s[i] != ' ' and i >= 0):
            i = i-1
        
        return j-i
        

File: Data Structures & Algorithms/non-overlapping-intervals/submission-2.cpp
class Solution {
public:
    int eraseOverlapIntervals(vector<vector<int>>& intervals) {
        sort(intervals.begin(), intervals.end(), [](const vector<int>& a, const vector<int>& b) {
            return a[1] < b[1];
        });
        int count = 0;
        int st = intervals[0][0];
        int end = intervals[0][1];
        int i = 1;
        while(i<intervals.size()) {        
            if(intervals[i][0]<end) {
                count++;
            } else {
                end = intervals[i][1];
            }
            i++;
        }
        return count;
    }
};
